src/test/streaming_test.cpp

```diff
@@ -1060,6 +1096,9 @@ node {
     }
 }
 )"};
+#ifdef _WIN32
+    replaceHandlerPath(testPbtxt);
```

Comment:
If we change replaceHandlerPath to return modified file:
```suggestion
    textPbtext = replaceHandlerPath(testPbtxt);
```

We can follow previous naming, like `getGenericFullPathForTmp`, for example:
// getGenericGraphPbtxt()

and then move if def inside it